lspSmithy/src/org/scala/abusers/lspsmithy/SmithyConverter.scala

```diff
@@ -93,11 +92,12 @@ object SmithyConverter:
             .filterNot(_.proposed)
             .filter {
               _.value match
-                case _: Int      => true
+                case _: Int      => false
                 case str: String => str.nonEmpty
             }
-            .map(convertEnumValues(shapeId, _))
-            .foldLeft(builder) { case (acc, item) => acc.addMember(item) }
+            .foldLeft(builder) { case (acc, entry) =>
+              acc.addMember(toUpperSnakeCase(entry.name.value), entry.value.stringValue)
```

Comment:
suggestion: imo if we're dealing with a mutable builder it's more natural to just do `foreach` but you do you

Reply:
i don't mind using foldLeft, it just feels more familiar

Reply:
kk

Reply:
I just feel like sometimes it can give you a false sense of security/immutability if you use mutable builders in an immutable fashion :)